modules/user_passwords/scripts/verify.ts

```diff
@@ -0,0 +1,44 @@
+import { Empty, RuntimeError, ScriptContext } from "../module.gen.ts";
+import { Algorithm, hashMatches } from "../utils/common.ts";
+
+export interface Request {
+	userId: string;
+    passGuess: string;
```

Comment:
Nit: not sure on the name passGuess, you're usually not guessing your own password

Reply:
Fair enough, maybe `passAttempt`?
This is the password that the user entered, so it's not necessarily correct.

Reply:
`passAttempt` works, or even just `password`. It might not be correct but its the password given by the user

Reply:
It's just `password` now!